src/components/Tooltip/Tooltip.mdx

```diff
@@ -96,6 +96,19 @@ on a disabled `Button`, pass the `shouldWrapChildren` prop.
 
 <Canvas of={TooltipStories.DisablingTooltipButton} />
 
+### Placement
+
+The `placement` prop adjusts where the tooltip is displayed relative to its children.
```

Comment:
List down some of the (commonly used) values you expect users to use.

Reply:
Or point to the "show code" button so they can see the values, but it's probably better to list them down explicitly.